paper/paper.bib

```diff
@@ -0,0 +1,113 @@
+    month = feb,
+    year = {2013},
+    pages = {314--319},
+}
+
+@misc{andersen2021optimade,
+      title={OPTIMADE: an API for exchanging materials data},
+      author={Casper W. Andersen and Rickard Armiento and Evgeny Blokhin and Gareth J. Conduit and Shyam Dwaraknath and Matthew L. Evans and Ádám Fekete and Abhijith Gopakumar and Saulius Gražulis and Andrius Merkys and Fawzi Mohamed and Corey Oses and Giovanni Pizzi and Gian-Marco Rignanese and Markus Scheidgen and Leopold Talirz and Cormac Toher and Donald Winston and Rossella Aversa and Kamal Choudhary and Pauline Colinet and Stefano Curtarolo and Davide Di Stefano and Claudia Draxl and Suleyman Er and Marco Esters and Marco Fornari and Matteo Giantomassi and Marco Govoni and Geoffroy Hautier and Vinay Hegde and Matthew K. Horton and Patrick Huck and Georg Huhs and Jens Hummelshøj and Ankit Kariryaa and Boris Kozinsky and Snehal Kumbhar and Mohan Liu and Nicola Marzari and Andrew J. Morris and Arash Mostofi and Kristin A. Persson and Guido Petretto and Thomas Purcell and Francesco Ricci and Frisco Rose and Matthias Scheffler and Daniel Speckhard and Martin Uhrin and Antanas Vaitkus and Pierre Villars and David Waroquiers and Chris Wolverton and Michael Wu and Xiaoyu Yang},
+      year={2021},
+      eprint={2103.02068},
+      archivePrefix={arXiv},
+      primaryClass={cond-mat.mtrl-sci}
+}
+
+@software{OPTIMADE_spec,
+  author       = {Andersen, Casper and
+                  Armiento, Rickard and
+                  Blokhin, Evgeny and
+                  Conduit, Gareth and
+                  Dwaraknath, Shyam and
+                  Evans, Matthew L and
+                  Fekete, Ádám and
+                  Gopakumar, Abhijith and
+                  Gražulis, Saulius and
+                  Merkys, Andrius and
+                  Mohamed, Fawzi and
+                  Oses, Corey and
+                  Pizzi, Giovanni and
+                  Rignanese, Gian-Marco and
+                  Scheidgen, Markus and
+                  Talirz, Leopold and
+                  Toher, Cormac and
+                  Winston, Donald},
+  title        = {The OPTIMADE Specification},
+  month        = jul,
+  year         = 2020,
+  publisher    = {Zenodo},
+  version      = {1.0},
+  doi          = {10.5281/zenodo.4195051},
+  url          = {https://doi.org/10.5281/zenodo.4195051}
+}
+
+@software{pydantic,
+    title = {pydantic},
+    version = {1.8.2},
+    url = {https://github.com/samuelcolvin/pydantic}
+}
+
+@software{FastAPI,
+    title = {FastAPI},
+    version = {0.65.1},
+    url = {https://github.com/tiangolo/fastapi}
+}
+
+@software{MongoDB,
+    title = {MongoDB},
+    version = {4.4},
+    url = {https://www.mongodb.com}
+}
+
+@misc{OpenAPI,
+    title = {The OpenAPI Specification},
+    version = {3.1},
+    url= {https://spec.openapis.org/oas/v3.1.0},
+}
```

Comment:
Not sure what the canonical form of the references here should be (they get rendered as inline "Author, year" refs in the paper) - perhaps these are better suited as in-line links if these packages do not provide blessed bibliography entries?

Reply:
@ml-evs I think a bib entry is preferred even for e.g. a github repo. From <https://joss.readthedocs.io/en/latest/submitting.html>:

```raw
If you want to cite a software repository URL (e.g. something on GitHub without a preferred
citation) then you can do it with the example BibTeX entry below for @fidgit.

...
 
@misc{fidgit,
  author = {A. M. Smith and K. Thaney and M. Hahnel},
  title = {Fidgit: An ungodly union of GitHub and Figshare},
  year = {2020},
  publisher = {GitHub},
  journal = {GitHub repository},
  url = {https://github.com/arfon/fidgit}
}
```